docs/src/quickstart.md

```diff
@@ -1,23 +1,17 @@
 # Quickstart
 
-This section describes how to test a PostgreSQL cluster on your laptop/computer
-using CloudNativePG on a local Kubernetes cluster in [Kind](https://kind.sigs.k8s.io/) or
+This section guides you through testing a PostgreSQL cluster on your local machine by deploying CloudNativePG on a local Kubernetes cluster using either [Kind](https://kind.sigs.k8s.io/) or
 [Minikube](https://kubernetes.io/docs/setup/learning-environment/minikube/).
 
-!!! Warning
-    The instructions contained in this section are for demonstration,
-    testing, and practice purposes only and must not be used in production.
+!!! Warning The instructions contained in this section are for demonstration, testing, and practice purposes only and must not be used in production.
```

Comment:
Sorry. No. Why are you changing the spacing and layout?
Again see https://squidfunk.github.io/mkdocs-material/reference/admonitions/
for the layout of admonition boxes.